### PR TITLE
Fix typo. Money(100, 'EUR') should be 100 euro's, not 1000.

### PR DESCRIPTION
**Pull request by WouterSioen**

### CODE REVIEW

**Comment by WouterSioen** [comment]
poke @liuggio

File: _posts/2014-03-21-persist-the-money-doctrine-value-object.md [hunk 1]
```diff
@@ -66,11 +66,11 @@ Well surely there is a more elegant way to do it, we are objects oriented progra
 With Doctrine minor2.5 a custom type is allowed only for one-to-one mapping between field and column of the relational database,
 and representing an object with more proprerties is difficult.
 
-A very common approach was to create a custom `MoneyType` see [Doctrine2/MoneyType](https://github.com/mathiasverraes/money/blob/708d8d53b2374e1f9686dceee4f9636df32f6d43/lib/Money/Doctrine2/MoneyType.php), with this custom type Money is converted to a string eg. `Money(100, 'EUR')` would become a `string` of `"1000 EUR"`.
```
Comment (liuggio) [inline _posts/2014-03-21-persist-the-money-doctrine-value-object.md]: I think that `Money(100, 'EUR')`  is 1.00 eur

**Comment by liuggio** [comment]
Thanks a lot!!!!! sorry for the missing reply, I'm a not polite :D

File: _posts/2014-03-21-persist-the-money-doctrine-value-object.md (hunk 1 above)
Comment (WouterSioen) [inline _posts/2014-03-21-persist-the-money-doctrine-value-object.md]: fixed

**Comment by WouterSioen** [comment]
No problem :)

**Comment by liuggio** [comment]
Wooo000ow you rock!!!

**Pull request merged**